examples/kyb-app/src/components/atoms/Stepper/components/atoms/Breadcrumb/base-theme.tsx

```diff
@@ -0,0 +1,47 @@
+import { CompletedIcon } from '@app/components/atoms/Stepper/components/atoms/Breadcrumb/components/CompletedIcon';
+import { ExclamationMarkIcon } from '@app/components/atoms/Stepper/components/atoms/Breadcrumb/components/ExclamationMarkIcon';
+import { BreadcrumbTheme } from '@app/components/atoms/Stepper/components/atoms/Breadcrumb/types';
+import { twMerge } from 'tailwind-merge';
+
```

Comment:
Reuse class name, extract common class name to a variable and replace this variable in all the relevant references 

```
const wrapperClassName = twMerge('box-border', 'w-[12px]', 'h-[12px]', 'rounded-full')
const outerClassName = twMerge('rounded-full', 'border-[2px] border-transparent')
```

and composite border with the base outer class for idle
```
    wrapper: {
      className: twMerge(wrapperClassName, 'border'),
      activeClassName: twMerge('border-[#007AFF]'),
    },
```